{"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/134/comments/36633868.json"}, "html": {"href": "#!/osrf/srcsim/issues/134#comment-36633868"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/srcsim/issues/134.json"}}, "type": "issue", "id": 134, "repository": {"links": {"self": {"href": "data/repositories/osrf/srcsim.json"}, "html": {"href": "#!/osrf/srcsim"}, "avatar": {"href": "data/bytebucket.org/ravatar/{c41021d9-0217-4628-999d-7f8d82f98dcb}ts=c_plus_plus"}}, "type": "repository", "name": "srcsim", "full_name": "osrf/srcsim", "uuid": "{c41021d9-0217-4628-999d-7f8d82f98dcb}"}, "title": "java.io.IOException: Too many open files"}, "content": {"raw": "Seeing the same 'Failed to accept a connection' errors here.  Unable to connect to any ros topics published by the robot at that point.  I had been running for 1:43:00 real time, 48:30:00 sim time.  I brought my system up and down about 20 times in that interval, so lots of subscribing.\n\n```\nMay 07, 2017 11:10:34 PM org.jboss.netty.channel.socket.nio.NioServerSocketPipelineSink\nWARNING: Failed to accept a connection.\njava.io.IOException: Too many open files\n\tat sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)\n\tat sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)\n\tat sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)\n\tat org.jboss.netty.channel.socket.nio.NioServerSocketPipelineSink$Boss.run(NioServerSocketPipelineSink.java:238)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)\n\tat java.lang.Thread.run(Thread.java:745)\n```", "markup": "markdown", "html": "<p>Seeing the same 'Failed to accept a connection' errors here.  Unable to connect to any ros topics published by the robot at that point.  I had been running for 1:43:00 real time, 48:30:00 sim time.  I brought my system up and down about 20 times in that interval, so lots of subscribing.</p>\n<div class=\"codehilite\"><pre><span></span>May 07, 2017 11:10:34 PM org.jboss.netty.channel.socket.nio.NioServerSocketPipelineSink\nWARNING: Failed to accept a connection.\njava.io.IOException: Too many open files\n    at sun.nio.ch.ServerSocketChannelImpl.accept0(Native Method)\n    at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:422)\n    at sun.nio.ch.ServerSocketChannelImpl.accept(ServerSocketChannelImpl.java:250)\n    at org.jboss.netty.channel.socket.nio.NioServerSocketPipelineSink$Boss.run(NioServerSocketPipelineSink.java:238)\n    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)\n    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)\n    at java.lang.Thread.run(Thread.java:745)\n</pre></div>", "type": "rendered"}, "created_on": "2017-05-08T03:16:29.056583+00:00", "user": {"display_name": "Steven Gray", "uuid": "{7470e809-c304-4b17-b986-685f31fa15c3}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B7470e809-c304-4b17-b986-685f31fa15c3%7D"}, "html": {"href": "https://bitbucket.org/%7B7470e809-c304-4b17-b986-685f31fa15c3%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/e6bee642b76507eb51c2839771722e68d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsSG-0.png"}}, "nickname": "stgray", "type": "user", "account_id": "557058:e6cc7c7b-9d1b-4abe-87d5-5f0da7c863e1"}, "updated_on": null, "type": "issue_comment", "id": 36633868}